docs/08_getting_started.md

```diff
@@ -0,0 +1,329 @@
+# Getting Started
+This section provides you with the steps that are required to use the extractor in your android app.
+
+
+## Configure project.
+### Add dependencies to gradle:
+
+1. **project** `settings.gradle`:
+
+```gradle
+...
+buildscript {
+    repositories {
+        ...
+        maven { url 'https://jitpack.io' }
+    }
+    ...
+}
+```
+
+
+
+2. module/app build.gradle:
+
+```gradle
+// NewPipe Libraries
+implementation 'com.github.TeamNewPipe:NewPipeExtractor:v0.21.15'
+implementation 'com.github.TeamNewPipe:nanojson:1d9e1aea9049fc9f85e68b43ba39fe7be1c1f751'
+
+
+// HTTP client
+implementation "com.squareup.okhttp3:okhttp:3.12.13"
+```
+
+Note that you could use any http client other then okhttp, but it used here for more simplicity.
+
+
+### Make sure you use Java 11
```

Comment:
Referencing: https://github.com/TeamNewPipe/documentation/pull/37/files#r784795350

Yes NewPipe does use Java 11, but that does not mean one can't use Java 8. You discovered it yourself that it is possible to use with java 8

I have a fork of NewPipe v0.19.x which was still based on Java 8 back then, where i just increment the extractor dependency